python/run-flink-sql.py

```diff
@@ -26,25 +26,39 @@ def execute_table_definitions(definitions: List[str], params: Dict[str, Any]) ->
     "--table-definition-path", nargs="+", required=True, help="Path to flink table DDL. Can be path to a file or s3"
 )
 parser.add_argument("--query", "-q", required=True, help="SQL query to execute.")
-parser.add_argument("--target-table", "-tt", required=True, help="Target table where to write results of the query.")
-parser.add_argument("--metadata-query-name", "-qname", required=True, help="Human readable SQL query name.")
-parser.add_argument("--metadata-query-description", "-qdesc", required=True, help="SQL query description.")
-parser.add_argument("--metadata-query-id", "-qid", required=True, help="Unique SQL query id.")
+parser.add_argument("--target-table", "-tt", required=False, help="Target table where to write results of the query.")
+parser.add_argument("--metadata-query-name", "-qname", required=False, help="Human readable SQL query name.")
+parser.add_argument("--metadata-query-description", "-qdesc", required=False, help="SQL query description.")
+parser.add_argument("--metadata-query-id", "-qid", required=False, help="Unique SQL query id.")
 parser.add_argument(
```

Comment:
Should not the arguments below be 1 line just like the rest? In jinja2 template these are 1 liners.

Reply:
`black` complains about wrong formatting. I'll change jinja template then.